jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -125,8 +125,8 @@ def get_all_code_review_comments(self, file_path_from_repo_root, git_root_dir):
 			review_comments = review.get("comments", [])
 			comments_on_file = []
 			for item in review_comments:
-				location = item["comment"]["location"]
-				if location.get("path", "") == file_path_from_repo_root:
+				location = item["comment"].get("location", "")
```

Comment:
Make the default here `{}` instead of `""`. That way, you can chain the `get` methods. You might also want to user `get` for indexing into `item` as well.

e.g. `path = item.get("comment", {}).get("location", {}).get("path", "")`